### ISSUE TITLE
Script job count is incorrect

### ISSUE THREAD

**Issue by DanSheps** [netbox-community/netbox#14572]
### Deployment Type

Self-hosted

### NetBox Version

v3.6.7

### Python Version

3.10

### Steps to Reproduce

1. Create a script file:

```python
from extras.scripts import Script


class TestScript(Script):
    name = 'Test'
    description = ''

    def run(self, data, commit=True):
        self.log_success('Finished Script')


class TestScriptNew(Script):
    name = 'Test New'
    description = ''

    def run(self, data, commit=True):
        self.log_success('Finished Script 2')
```
2. Upload to NetBox
3. Run `Test`
5. View `Test New`

### Expected Behavior

Job tab badge has "0"

### Observed Behavior

Job tab badge has >0

**Comment by DanSheps** [comment]
This is caused by the badge querying `module.jobs` without filtering for the script name.

**Comment by DanSheps** [comment]
Was fixed by #14199

**Comment by DanSheps** [comment]
Wait, something different I think
